commitizen/bump.py

```diff
@@ -45,7 +45,12 @@ def find_increment(
 
 
 def update_version_in_files(
-    current_version: str, new_version: str, files: list[str], *, check_consistency=False
+    current_version: str,
+    new_version: str,
+    files: list[str],
+    encoding: str,
+    *,
```

Comment:
If possible, I'd like to keep newly added argument as keyword only argument
```suggestion
    *,
    encoding: str,
```

Reply:
Resolved in commit `e9d73c5f`.